pkg/frontend/sku_test.go

```diff
@@ -1,4 +1,4 @@
-package dynamic
+package frontend
```

Comment:
could you remove the target location field from the test fields? it is never actually used

Reply:
Removed target location tests fields